src/plugins/data/public/autocomplete/autocomplete_service.ts

```diff
@@ -63,7 +61,7 @@ export class AutocompleteService {
     const provider = this.querySuggestionProviders.get(language);
 
     if (provider) {
-      return provider({ core: this.core, ...args });
+      return provider(args);
```

Comment:
is this a breaking change?

Reply:
no, i'm reverting a change that i made previously with dql autocomplete. this was before i knew about a clear way to get to the http core util. but now since we're using services, its redundant and so i put it back the way it was.